monitoring/dashboards/application.yaml

```diff
@@ -10,1442 +10,1697 @@ metadata:
     app: observability-grafana
 data:
  applications.json: |-
-    {
-      "annotations": {
-        "list": [
-          {
-            "builtIn": 1,
-            "datasource": {
-              "type": "datasource",
-              "uid": "grafana"
-            },
-            "enable": true,
-            "hide": true,
-            "iconColor": "rgba(0, 211, 255, 1)",
-            "name": "Annotations & Alerts",
-            "type": "dashboard"
-          }
-        ]
-      },
-      "description": "Istio Mesh Dashboard version 1.17.8",
-      "editable": true,
-      "fiscalYearStartMonth": 0,
-      "gnetId": 7639,
-      "graphTooltip": 0,
-      "id": 29,
-      "links": [],
-      "liveNow": false,
-      "panels": [
+  {
+    "annotations": {
+      "list": [
         {
+          "builtIn": 1,
           "datasource": {
-            "type": "prometheus",
-            "uid": "Prometheus"
+            "type": "datasource",
```

Comment:
Why is the datasource called grafana?

Reply:
The git diff is throwing a wrong idea about the changes, this grafana datasource has not been replaced in place of prometheus, It has always been there as default with any grafana dashboard. 

For reference, here is a [sample grafana dashboard](https://play.grafana.org/d/T512JVH7z/loki-nginx-service-mesh-json-version?orgId=1&editview=dashboard_json). You can see the grafana datasource defined in the beginning.

Also I have not made any changes to non Loki things, these were already present when I picked up. I used to make changes to the dashboard and then download the updated dashboard json from the UI and paste it in the configmap.